spring-boot/admin/src/main/resources/templates/admin/fragments/api-keys/list.html

```diff
@@ -237,5 +175,15 @@ <h2 th:text="#{admin.api-keys.subtitle}">[Post Administration</h2>
         </div>
         <div th:include="admin/fragments/scripts :: scripts" th:remove="tag">Javascript Libraries</div>
         <div th:include="admin/fragments/scripts :: initializeDataTables" th:remove="tag">Javascript Libraries</div>
+        <div th:include="admin/fragments/scripts :: apiKeyRemoveDialog" th:remove="tag">Javascript Libraries</div>
+
+    <script>
+        function createInfoBoxMessageAndDeleteKey(apiKeyId, consumerName){
+            infoBoxMessage = "Are you sure you want to delete the API key of <b>" + consumerName +
```

Comment:
suggestion: `askForConfirmationMessage`

Reply:
I found ```createWarningBoxAndAskForPermissionToDeleteApiKey``` to be descriptive and implemented it. I think this can considered to be done.

Reply:
As it actually deletes the key, see my other comment about naming suggestions.